Detectors/Calibration/src/MeanVertexCalibrator.cxx

```diff
@@ -201,21 +211,18 @@ void MeanVertexCalibrator::fitMeanVertex(o2::calibration::MeanVertexData* c, Mea
 
   // now we update the error on x
   double sumX = 0, sumY = 0, weightSumX = 0, weightSumY = 0;
-  for (int iFit = 0; iFit < mNPointsForSlope; ++iFit) {
+  for (int iFit = 0; iFit < nBinsOK; ++iFit) {
```

Comment:
Should we check somewhere if `nBinsOk` is > `mNPointsForSlope`?

Reply:
It cannot be ` >mNPointsForSlope`, it is incremented in the loop `0:mNPointsForSlope` for successful fits.

Reply:
I expressed myself wrongly: what if we have less nBinsOk than the needed number to calculate the slope?

Reply:
`nBinsOK < 2` leads to failed fit

Reply:
ok, thanks.